py3status/modules/file_status.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Display if a file or dir exists.
+
+Configuration parameters:
+    cache_timeout: how often to run the check
+    format_available: what to display when available
+    format_unavailable: what to display when unavailable
+    path: check if file or dir exists
+
+@author obb, Moritz Lüdecke
+"""
+
+from time import time
+import os
+import subprocess
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 10
+    format_available = u'●'
```

Comment:
totally arbitrary comment, do we feel a bullet point is a good default to state if a file is present or not? I'm not sure tbh